libs/ngxtension/create-notifier/src/create-notifier.ts

```diff
@@ -0,0 +1,12 @@
+import { signal } from '@angular/core';
+
+export function createNotifier() {
```

Comment:
`createNotifier()` is fine, but I'd give more context to the name like I mentioned on the discussion, like: 

`createManualSignalNotifier()` or just `createSignalNotifier()`.

Just a matter of preference though, `createNotifier()` is also fine.

Reply:
Happy to see what more people think but personally I find `createManualSignalNotifier()` too verbose - and I suppose if you do prefer being more explicit in the code you could still make the notifier names themselves more verbose e.g: `myManualSignalNotifier = createNotifier()`